python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -38,6 +44,34 @@
 from lsst.pipe.base import AlgorithmError
 
 
+class PsfexNoStarsError(AlgorithmError):
+    """Raised if no stars are available for PSF determination."""
+
+    def __init__(self) -> None:
+        super().__init__("No psf candidates supplied.")
+
+
+class PsfexNoGoodStarsError(AlgorithmError):
+    """Raised if no "good" stars are available for PSF determination.
+
+    Parameters
+    ----------
+    num_available_stars : `int`
+        Number of available stars for PSF determination.
+    """
+
+    def __init__(self, num_available_stars) -> None:
+        self._num_available_stars = num_available_stars
+        super().__init__("No good psf candidates to pass to psfex.")
```

Comment:
Oh, I think this message should include the one value that *is* passed in here:

```suggestion
        self._num_available_stars = num_available_stars
        super().__init__("No good psf candidates out of {num_available_stars} candidates.")
```

Reply:
Good point, done!